pages/reminder-pledge.vue

```diff
@@ -2,14 +2,26 @@
   <div class="page">
     <div class="bg-white">
       <div class="page-fade-in pt-28">
-        <ThanksSection
-          title="Thanks! We’ll be in touch again when it’s time to move your money out of fossil banking."
-        />
+        <div v-if="reminderpledge?.data.slices">
+          <SliceZone
+            :slices="reminderpledge?.data.slices ?? []"
+            :components="sliceComps"
+          />
+        </div>
+        <div v-else>
+          <h1>Thanks! We’ll be in touch again when it’s time to move your money out of fossil banking.</h1>
```

Comment:
Should just use the ThanksSection here (the h1 ends up misaligned and styling is ignored): 

```vue
<ThanksSection
    title="Thanks! We’ll be in touch again when it’s time to move your money out of fossil banking."
 />
```